doc/index.rst

```diff
@@ -205,13 +205,13 @@
                     <h4>News</h4>
                     <ul>
                     <li><em>On-going development:</em>
-                    <a href="whats_new.html"><em>What's new</em> (Changelog)</a>
+                    <a href="/dev/whats_new.html"><em>What's new</em> (Changelog)</a>
```

Comment:
The anchor tags should work fine, but I doubt this change is going to work.
I have no suggestion though.